reversion_compare/mixins.py

```diff
@@ -223,10 +223,12 @@ def compare_DateTimeField(self, obj_compare):
         return render_to_string("reversion-compare/compare_DateTimeField.html", context)
 
     def compare_BooleanField(self, obj_compare):
-        ''' compare all model BooleanField field '''
+        ''' compare booleans as a complete field, rather than as a string '''
         context = {
-            "bool1": obj_compare.value1,
-            "bool2": obj_compare.value2,
+            "bool1": str(obj_compare.value1),
+            "bool2": str(obj_compare.value2),
         }
         return render_to_string("reversion-compare/compare_BooleanField.html", context)
-    compare_NullBooleanField=compare_BooleanField
\ No newline at end of file
+
+    def compare_NullBooleanField(self, obj_compare):
+        return self.compare_BooleanField(obj_compare)
```

Comment:
This is a stylistic change, I prefer explicit methods rather than method properties - its a personal preference, but I find it clearer than

    compare_NullBooleanField = compare_BooleanField

But am prepared to revert.